.github/release/standard/Dockerfile

```diff
@@ -3,11 +3,16 @@ FROM python:3.8.8-slim
 ENV DEBIAN_FRONTEND noninteractive
 ENV TERM linux
 
+RUN apt-get update
+
+# Install compiler tools
+RUN apt-get --yes install python-dev build-essential gcc
```

Comment:
This will probably not be required, see https://github.com/earthobservations/wetterdienst/pull/407/files#r607366233.